src/component-select/component-select.js

```diff
@@ -68,7 +68,7 @@ export default () => {
           ) : (
             <>
               <span>{selectedPage}</span>
-              <button onClick={editPageName}>&#x270E;</button>
+              <button className="edit-page-name" onClick={editPageName}><i class="fas fa-pen"></i></button>
```

Comment:
Do you like the font-awesome icons better than the Unicode characters for similar things?

Reply:
I do, yes. They give you a lot more control regarding icon options, and also styling.